src/library/scala/Option.scala

```diff
@@ -201,12 +272,27 @@ sealed abstract class Option[+A] extends Product with Serializable {
   /** Returns this $option if it is nonempty '''and''' applying the predicate $p to
    * this $option's value returns false. Otherwise, return $none.
    *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(x) if !p(x) => Some(x)
+   *   case _                => None
+   * }
+   * }}}
```

Comment:
You can mention that this is also equivalent to `option.filter( x => ! p(x) )`

Reply:
Yes, that's also true, but is orthogonal to what I'm proposing.